cmd/configure/devicetest.go

```diff
@@ -0,0 +1,139 @@
+package configure
+
+import (
+	"errors"
+	"time"
+
+	"github.com/evcc-io/evcc/api"
+	"github.com/evcc-io/evcc/charger"
+	"github.com/evcc-io/evcc/meter"
+	"github.com/evcc-io/evcc/util/templates"
+	"github.com/evcc-io/evcc/vehicle"
+	"gopkg.in/yaml.v3"
+)
+
+type DeviceTestResult string
+
+const (
+	DeviceTestResult_Valid              DeviceTestResult = "Valid"
```

Comment:
die _ sind immer wenn es übersetzte Strings sind? Sonst in Go eher unüblich, find ich aber zur Unterscheidung gut.

Reply:
nein, wird nicht übersetzt. Habs umbenannt